test/stdlib/PrintFloat.swift.gyb

```diff
@@ -623,11 +623,11 @@ PrintTests.test("Printable_Float16") {
   expectNaN("nan(0xff)", Float16(bitPattern: 0x7eff))
   expectNaN("-nan", -Float16.nan)
   expectNaN("-nan(0xff)", -Float16(nan: 255, signaling: false))
-  expectNaN("snan", Float16.signalingNaN)
-  expectNaN("-snan", -Float16.signalingNaN)
-  expectNaN("snan(0xff)", Float16(nan: 255, signaling: true))
-  expectNaN("-snan(0xff)", -Float16(nan: 255, signaling: true))
-  expectNaN("snan(0xff)", Float16(bitPattern: 0x7dff))
+  expectFailure { expectNaN("snan", Float16.signalingNaN) }
+  expectFailure { expectNaN("-snan", -Float16.signalingNaN) }
+  expectFailure { expectNaN("snan(0xff)", Float16(nan: 255, signaling: true)) }
+  expectFailure { expectNaN("-snan(0xff)", -Float16(nan: 255, signaling: true)) }
+  expectFailure { expectNaN("snan(0xff)", Float16(bitPattern: 0x7dff)) }
```

Comment:
Please add a couple additional test cases to make sure these still print as nan.

Reply:
Done.